seq2science/rules/quantification.smk

```diff
@@ -298,14 +298,15 @@ elif config["quantifier"] == "kallistobus":
                 mem_gb=66,
             params:
                 basename=lambda wildcards, input: f"{input.basedir[0]}/{wildcards.assembly}",
+                barcode_arg=lambda wildcards, input: "-w " + input.barcodefile[0] if input.barcodefile else "", 
```

Comment:
again, not sure if I'm right, but if `False`, wont `"-w " + input.barcodefile[0] if input.barcodefile else ""` give `-w `?
if so, solution is  `("-w " + input.barcodefile[0]) if input.barcodefile else ""`

Reply:
If that expression is true, it returns the complete part before the if and "" otherwise. At least the command line output did not contain an empty `-w`. However, putting it in braces makes it a bit more readable.